stage1/init/init.go

```diff
@@ -171,6 +172,50 @@ func machinedRegister() bool {
 	return found == 2
 }
 
+func lookupPath(bin string, paths string) (string, error) {
+	pathsArr := strings.Split(paths, ":")
+	for _, path := range pathsArr {
+		binPath := filepath.Join(path, bin)
+		binAbsPath, err := filepath.Abs(binPath)
+		if err != nil {
+			return "", fmt.Errorf("unable to find absolute path for %s", binPath)
+		}
+		if _, err := os.Stat(binAbsPath); err == nil {
+			return binAbsPath, nil
+		}
+	}
+	return "", fmt.Errorf("unable to find %q in %q", bin, paths)
+}
+
+func installAssets() error {
+	systemctlBin, err := lookupPath("systemctl", os.Getenv("PATH"))
+	if err != nil {
+		return err
+	}
+	bashBin, err := lookupPath("bash", os.Getenv("PATH"))
+	if err != nil {
+		return err
+	}
+	systemdShutdownBin, err := lookupPath("systemd-shutdown", "/lib/systemd:/usr/lib/systemd")
+	if err != nil {
+		return err
+	}
+	systemdBin, err := lookupPath("systemd", "/lib/systemd:/usr/lib/systemd")
+	if err != nil {
+		return err
+	}
+
+	assets := []string{}
+	assets = append(assets, "/lib/systemd/systemd:"+systemdBin)
```

Comment:
You could use proj2aci.GetAssetString() instead (https://github.com/appc/goaci/blob/f99814ac2ac967bc202afd1c923cde30a65811ed/proj2aci/asset.go#L13).